stacks/api/main.ts

```diff
@@ -1,23 +1,184 @@
+import {Module} from "@nestjs/common";
 import {NestFactory} from "@nestjs/core";
+import {ActivityModule} from "@spica-server/activity";
+import {BucketModule} from "@spica-server/bucket";
 import {Middlewares} from "@spica-server/core";
+import {SchemaModule} from "@spica-server/core/schema";
+import {CREATED_AT, DATE_TIME, OBJECT_ID, UPDATED_AT} from "@spica-server/core/schema/defaults";
+import {DashboardModule} from "@spica-server/dashboard";
+import {DatabaseModule} from "@spica-server/database";
+import {FunctionModule} from "@spica-server/function";
+import {PassportModule} from "@spica-server/passport";
+import {PreferenceModule} from "@spica-server/preference";
+import {StorageModule} from "@spica-server/storage";
 import * as fs from "fs";
 import * as https from "https";
-import {AppModule} from "./app.module";
+import * as yargs from "yargs";
 
-async function bootstrap() {
-  let httpsOptions: https.ServerOptions;
+const args = yargs
+  /* TLS Options */
+  .options({
+    "cert-file": {
+      string: true,
+      description: "Path to the client server TLS cert file."
+    },
+    "key-file": {
+      string: true,
+      description: "Path to the client server TLS key file."
+    }
+  })
+  /* Database Options */
+  .options({
+    "database-uri": {
+      string: true,
+      description: "MongoDB connection url."
+    },
+    "database-name": {
+      string: true,
+      description: "Name of the database."
+    },
+    "database-replica-set": {
+      string: true,
+      alias: ["replica-set"],
+      description: "Name of the replica set."
+    },
+    "database-pool-size": {
+      number: true,
+      description: "Amount of connection that will be opened against database.",
+      default: 100

```

Comment:
It was supposed to be 10, right?

Reply:
I didn't notice that 👍 Let me change that